eprihlaska/validators.py

```diff
@@ -25,14 +25,15 @@ def __call__(self, form, field):
         bno_reg = re.compile(r'^([0-9][0-9][0156][0-9][0-3][0-9])/(\d\d\d\d)$')
         mo = bno_reg.search(field.data)
 
-        try:
-            mo.group(0)
-        except AttributeError:
-            raise validators.ValidationError(self.message + ' ' + self.form_err)
-
-        birth_no = int(mo.group(1) + mo.group(2))
-        if birth_no % 11 != 0:
-            raise validators.ValidationError(self.message + ' ' + self.mod_err)
+        if field.data is not '':
```

Comment:
@JakubNvk Mozeme sem dat kratky komentar k tejto logike? Teda taky, co by vysvetloval, ze je to preto, lebo aj prazdne rodne cislo je akceptovane (ak ho niekto nema priradene), ale ak sa uz rodne cislo vyplna, tak by malo byt spravne.

Reply:
@mrshu jasne, pridam.

Reply:
@JakubNvk Este mala poznamka: tato podmienka by asi mohla byt aj iba `if field.data`, co by zrejme trochu pomohlo citatelnosti.